gateway/src/constructors/express/routes/index.js

```diff
@@ -1,30 +1,50 @@
-const path = require('path')
-const cors = require('cors')
-const router = require('express').Router()
+const cors = require("cors");
+const router = require("express").Router();
+const {  query, validationResult } = require("express-validator");
 
-module.exports = (context = {}) => {
-  const APIRouter = require('./api')(context)
+module.exports = ({sequelize, models,...context }) => {
+ const APIRouter = require("./api")(context);
+ const PUBLICRouter = require("./public")(context);
```

Comment:
Perhaps we should call it `PublicRouter`, with no need to have full capitals for PUBLIC